embedded/src/main/java/org/wildfly/core/embedded/EmbeddedServerFactory.java

```diff
@@ -71,17 +69,53 @@
     private static final String SYSPROP_KEY_LOGMANAGER = "java.util.logging.manager";
     private static final String SYSPROP_KEY_JBOSS_HOME_DIR = "jboss.home.dir";
     private static final String SYSPROP_KEY_JBOSS_MODULES_DIR = "jboss.modules.dir";
-    private static final String SYSPROP_KEY_JBOSS_BUNDLES_DIR = "jboss.bundles.dir";
     private static final String SYSPROP_VALUE_JBOSS_LOGMANAGER = "org.jboss.logmanager.LogManager";
 
     private EmbeddedServerFactory() {
     }
 
-    public static StandaloneServer create(String jbossHomePath, String modulePath, String bundlePath, String... systemPackages) {
-        return create(jbossHomePath, modulePath, bundlePath, systemPackages, null);
+    /**
+     * Create an embedded standalone server with modules loaded from the standard location in the server installation.
+     *
+     * @param jbossHomePath the location of the root of server installation. Cannot be {@code null} or empty.
+     * @param systemPackages names of any packages that must be treated as system packages, with the same classes
+     *                       visible to the caller's classloader visible to server-side classes loaded from
+     *                       the server's modular classlaoder
+     * @return the server. Will not be {@code null}
+     */
+    public static StandaloneServer create(String jbossHomePath, String... systemPackages) {
```

Comment:
This one should probably be removed. Using two String parameters here will result in the `public static StandaloneServer create(String jbossHomePath, String modulePath, String... systemPackages` being invoked.